qml/components/settingsPage/SettingsBehavior.qml

```diff
@@ -135,6 +135,17 @@ AccordionItem {
                 }
             }
 
+            TextSwitch {
```

Comment:
Shouldn't it also be hidden if `notificationFeedback` is set to `NotificationFeedbackNone`, like the switch below?